apps/AmplifyDemo/lambda/index.js

```diff
@@ -0,0 +1,135 @@
+    var r = Math.random() * 16 | 0, v = c === 'x' ? r : (r & 0x3 | 0x8);
+    return v.toString(16);
+  });
+}
+
+function response(statusCode, contentType, body) {
+  return {
+    statusCode: statusCode,
+    headers: { 'Content-Type': contentType },
+    body: body,
+    isBase64Encoded: false
+  };
+}
+
+const createChimeMeeting = async (context) => {
+  const title = context.arguments.title;
+  const region = context.arguments.region || 'us-east-1';
+  const name = context.arguments.name;
+
+  if (!title || !name) {
+    return response(400, 'application/json', JSON.stringify({
+      error: 'Required properties: meeting title, name'
+    }));
+  }
+  const request = {
+    ClientRequestToken: uuid(),
+    MediaRegion: region,
+    NotificationsConfiguration: USE_EVENT_BRIDGE === 'false' ? { SqsQueueArn: SQS_QUEUE_ARN } : {},
+    
+    // Any meeting ID you wish to associate with the meeting.
+    // For simplicity here, we use the meeting title.
+    ExternalMeetingId: title.substring(0, 64),
+    Tags: [
+      { Key: 'Department', Value: 'RND'}
+    ]
+  };
+  console.info('Creating new chime meeting: ' + JSON.stringify(request));
+  meetingInfo = await chime.createMeeting(request).promise();
+
+  console.info('Adding new attendee');
+  const attendeeInfo = (await chime.createAttendee({
+    MeetingId: meetingInfo.Meeting.MeetingId,
+
+    // Any user ID you wish to associate with the attendeee.
+    // For simplicity here, we use a random UUID for uniqueness
+    // combined with the name the user provided, which can later
+    // be used to help build the roster.
+    ExternalUserId: `${uuid().substring(0, 8)}#${name}`.substring(0, 64),
+  }).promise());
+
+  return response(200, 'application/json', JSON.stringify(
+    {
+      Meeting: meetingInfo.Meeting,
+      Attendee: attendeeInfo.Attendee,
+    }, null, 2));
+}
+
+const joinChimeMeeting = async (context) => {
+  const meetingId = context.arguments.meetingId;
+  const name = context.arguments.name;
+
+  if (!meetingId || !name) {
+    return response(400, 'application/json', JSON.stringify({
+      error: 'Required properties: meeting Id, name'
+    }));
+  }
+
+  console.info('Adding new attendee');
+  const attendeeInfo = (await chime.createAttendee({
+    MeetingId: meetingId,
+
+    // Any user ID you wish to associate with the attendeee.
+    // For simplicity here, we use a random UUID for uniqueness
+    // combined with the name the user provided, which can later
+    // be used to help build the roster.
+    ExternalUserId: `${uuid().substring(0, 8)}#${name}`.substring(0, 64),
+  }).promise());
+
+  return response(200, 'application/json', JSON.stringify(
+    {
+      Attendee: attendeeInfo.Attendee
+    }, null, 2));
+}
+
+const endChimeMeeting = async (context) => {
+  const meetingId = context.arguments.meetingId;
+  await chime.deleteMeeting({ MeetingId: meetingId });
+  console.log("Deleted Meeting: " + meetingId);
```

Comment:
nit: Use single quotes for consistency with other strings. Or you can run ESLint or Formatter once to fix all minor syntaxes like missing semicolon for an arrow function.